tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = linters
```

Comment:
```suggestion
envlist = ruff, darglint, pylint
```

Reply:
Changed this to individual envs instead of all calling inside the linters env because if one command has a non-zero exit, tox exits and does not run the rest of the commands.